papermill/tests/test_execute.py

```diff
@@ -9,12 +9,46 @@
 import nbformat
 
 from ..api import read_notebook
-from ..execute import execute_notebook, log_outputs
+from ..execute import execute_notebook, log_outputs, _form_escaped_python_value
 from ..exceptions import PapermillExecutionError
 from . import get_notebook_path, RedirectOutput
 
 python_2 = sys.version_info[0] == 2
 
+class TestPythonEscapers(unittest.TestCase):
```

Comment:
I believe you could refactor this to one test case with parameters using pytest.

Reply:
Hmm true, seems odd to have lots of mixed pytest and unittest components. Are we planning on converting everything over to pytest or just using whichever is cleaner for particular tests? I'll convert this one, just curious on overall direction you see.

Reply:
My vote is pytest all the way.

Reply:
@MSeal The preferred approach for new tests would be to use pytest as it is far less verbose and maintainable than unittest. The pytest runner will collect tests from either so we don't need to convert from unittest for existing tests. A good project with very clean tests is `requests`.